docs/4.1. Behaviour - Registration.md

```diff
@@ -81,13 +81,13 @@ The following error conditions describe likely scenarios encountered by Nodes wh
 
 ### Node Encounters HTTP 200 On First Registration
 
-If a Node is restarted, its first action upon discovering a Registration API is to register its 'Node' resource. On first registration with a Registration API this SHOULD result in a '201 Created' HTTP response code. If a Node receives a 200 code in this case, a previous record of the Node can be assumed to still exist within the network registry. In order to avoid the registry-held representation of the Node's resources from being out of sync with the Node's view, an HTTP DELETE SHOULD be performed in this situation to explicitly clear the registry of the Node and any sub-resources. A new Node registration after this point SHOULD result in the correct 201 response code.
+If a Node is restarted, its first action upon discovering a Registration API is to register its 'Node' resource. On first registration with a Registration API this SHOULD result in a '201 Created' HTTP response code. If a Node receives a 200 code in this case, a previous record of the Node can be assumed to still exist within the network registry. In order to avoid the registry-held representation of the Node's resources from being out of sync with the Node's view, an HTTP `DELETE` SHOULD be performed in this situation to explicitly clear the registry of the Node and any sub-resources. A new Node registration after this point SHOULD result in the correct 201 response code.
 
 ### Node Encounters HTTP 400 (Or Other Undocumented 4xx) On Registration
 
 A 400 error indicates a client error which is likely to be the result of a validation failure identified by the Registration API. The same request MUST NOT be re-attempted without corrective action being taken first.
 
-A registry MAY issue a 400 code from the `/resource` POST endpoint for various reasons, including the following cases. It is not expected that a Node will be able to automatically handle these errors and user intervention could be necessary.
+A registry MAY issue a 400 code from the `/resource` `POST` endpoint for various reasons, including the following cases. It is not expected that a Node will be able to automatically handle these errors and user intervention could be necessary.
```

Comment:
Again here and below, is "400" an integer or a message fragment?

Reply:
Let's discuss